third_party/ros/roslaunch/BUILD.bazel

```diff
@@ -1,21 +1,36 @@
 load("@rules_python//python:defs.bzl", "py_binary", "py_library")
 load("@rules_ros_pip_deps//:requirements.bzl", "requirement")
 load("//build_tools:pylint.bzl", "pylint")
-load("//third_party:expand_template.bzl", "expand_template")
 
-expand_template(
+genrule(
     name = "deps",
-    out = "deps.py",
-    data = ["roscore.xml"],
-    substitutions = {"{roscore_xml_path}": "$(rootpath roscore.xml)"},
-    template = "deps.py.tpl",
+    outs = ["deps.py"],
+    cmd = """cat <<EOF > $@
+ROSMASTER_PATH = '$(rootpath @ros_comm//:rosmaster)'
```

Comment:
Unfortunately, you can't do rootpath on py_binary (as it has >1 files in defaultinfo, known issue). You need to wrap a binary in a symlink (feel free to copy https://github.com/mvukov/rules_ros2/blob/main/third_party/symlink.bzl) and then do rootpath on the symlink target.

Reply:
When I cherry pick this branch to my `bzl-mod` branch, this is what I get:
```
$ cat bazel-bin/third_party/ros/roslaunch/deps.py
ROSMASTER_PATH = 'external/_main~non_module_dependencies~ros_comm/rosmaster'
ROSCORE_XML_PATH = 'third_party/ros/roslaunch/roscore.xml'
```

Is that correct?

Reply:
I think this patch works as far as I can see. Have you had a chance to test it out?

Reply:
Sorry, it was a bit busy here. I just left some small comments.